index.html

```diff
@@ -3,17 +3,22 @@
     <head>
         <title>Axit</title>
         <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
     </head>
     <body>
         <header class="header">
-            <img src="/programming/web/projects/Axitproject/images/logo.png" alt="Logo">
+            <img src="images/logo.png" alt="Logo">
 
             <nav class="nav">
-                <a class="nav_link">Features</a>
-                <a class="nav_link">About</a>
-                <a class="nav_link">Pricing</a>
-                <a class="nav_link">Reviews</a>
-                <a class="nav_link">Contact</a>
+                <ul class="ul_link">
+                    <li class="li_link"><a class="nav_link" href="#">Features</a></li>
```

Comment:
It's OK when `<a>` wraps `<li>`. I think it will be more logical in your case. Because as for now your `<li>` is smaller then `<a>` which is its child :)
So I suggest to make the following structure: `<ul>` => `<a>` => `<li>`.